src/ILCompiler.CppCodeGen/src/CppCodeGen/ILToCppImporter.cs

```diff
@@ -185,6 +199,35 @@ public void SetParameterNames(IEnumerable<string> parameters)
             _parameterIndexToNameMap = parameterIndexToNameMap;
         }
 
+        private ISymbolNode GetGenericLookupHelper(ReadyToRunHelperId helperId, object helperArgument)
+        {
+            if (_method.RequiresInstMethodDescArg())
+            {
+                return _nodeFactory.ReadyToRunHelperFromDictionaryLookup(helperId, helperArgument, _method);
+            }
+            else
+            {
+                Debug.Assert(_method.RequiresInstArg() || _method.AcquiresInstMethodTableFromThis());
```

Comment:
>RequiresInstArg [](start = 37, length = 15)

`RequiresInstMethodTableArg` matches more closely what we need to assert than `RequiresInstArg`

Reply:
Thank you!